### PR TITLE
scripts: update image builder

### PR DESCRIPTION
**Pull request by HaokunX-intel**

### CODE REVIEW

File: scripts/guest-image/scripts/install_ai_workload.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DIR='/root/example_ai_workload'
+
+apt install python3-pip
```
Comment (kenplusplus) [inline scripts/guest-image/scripts/install_ai_workload.sh]: add -y

File: scripts/guest-image/scripts/run_ai_workload.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+BENCH_DIR='models/benchmarks'
+
+python3 -m pip install intel-tensorflow-avx512==2.11.0 numpy google-api-python-client tensorflow
+
+pushd $DIR/$BENCH_DIR
+
+if [[ $1 == "avx"  ]]; then
+    export DNNL_MAX_CPU_ISA=AVX512_CORE
+    echo "============"
+    echo "AVX Only!"
```
Comment (kenplusplus) [inline scripts/guest-image/scripts/run_ai_workload.sh]: what is AMX configure?

File: scripts/guest-image/scripts/run_ai_workload.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    echo "============"
+fi
+export OMP_NUM_THREADS=16
```
Comment (kenplusplus) [inline scripts/guest-image/scripts/run_ai_workload.sh]: could we make this script robust to accept the parameter of threads?

File: scripts/guest-image/scripts/run_ai_workload.sh (hunk 2 above)
Comment (HaokunX-intel) [reply scripts/guest-image/scripts/run_ai_workload.sh]: The default config is to enable `AMX`, equal to `export DNNL_MAX_CPU_ISA=AVX512_CORE_AMX`.

`export DNNL_MAX_CPU_ISA=AVX512_CORE` disables the `AMX` feature, for comparation with `AMX` feature enabled.

File: scripts/guest-image/scripts/install_ai_workload.sh (hunk 1 above)
Comment (HaokunX-intel) [reply scripts/guest-image/scripts/install_ai_workload.sh]: Sure!

File: scripts/guest-image/scripts/run_ai_workload.sh (hunk 3 above)
Comment (HaokunX-intel) [reply scripts/guest-image/scripts/run_ai_workload.sh]: Sure. Expose the parameter as an option with default value 16.

File: scripts/guest-image/scripts/run_ai_workload.sh [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+BENCH_DIR='models/benchmarks'
+
+THREADS=16
+AVX_ONLY=false
+
+usage() {
+    cat << EOM
+Usage: $(basename "$0") [OPTION]...
+  -a			    Enable AVX only, disable AMX
+  -t 			    The OMP thread number, 16 default
```
Comment (kenplusplus) [inline scripts/guest-image/scripts/run_ai_workload.sh]: -t ==> "-t <thread_count>"

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**